Cargo.toml

```diff
@@ -13,7 +13,9 @@ categories = ["network-programming", "game-development"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+default = ["serde"]
```

Comment:
Perhaps not enable it by default? Does bevy depend on serde or postcard?

Reply:
Good point! I've had a look through `bevy`, `matchbox` (with `ggrs` feature enabled), and `ggrs`. Out of those three crates, `serde` version 1 is always included, `ron` version 0.8 is included (from `bevy`), and `bincode` is included (`matchbox` and `ggrs`)

I've updated this PR to use `ron` and `bincode` instead, as they are already in the dependency graph. I've still kept them behind feature flags so you can remove them if desired.